packages/reaction-core/client/main.js

```diff
@@ -141,8 +141,13 @@ _.extend(ReactionCore, {
    * @returns {String} Session "admin/showActionView"
    */
   showActionView: function (viewData) {
-    Session.set("admin/showActionView", true);
-    ReactionCore.setActionView(viewData);
+    if (Session.get("admin/showActionView") === true && viewData &&
+      JSON.stringify(Session.get("admin/actionView")) === JSON.stringify(viewData)) {
+        Session.set("admin/showActionView", false);
+    } else {
+      Session.set("admin/showActionView", true);
+      ReactionCore.setActionView(viewData);
+    }
```

Comment:
It should not automatically close if you open the same action view. It should stay open unless you explicitly close the panel, or navigate away from the page or view. This is because the current buttons that open Action Views do not indicate that by clicking them again will close them.

In the future when theres a need to do a toggle switch for the Action View, then we will implement something like this with all the required UI updates.